dbmanager.py

```diff
@@ -8,7 +8,3 @@ def startUp():
 # function to close an instance of the database
 def stop():
     os.system("mysql.server stop")
-
-# function to check if the database exist, if not then create one
-def verifyTables(dnName):
```

Comment:
No it is not. I was writing it to test if tables existed in the database, when I then realized Dane's code already did that. I must've for got to remove it, but I did remove it in my latest push.